packages/client-core/src/common/components/LazyImage/index.tsx

```diff
@@ -57,6 +57,7 @@ export const LazyImage = ({ src, alt }) => {
     <CardMedia
       component="img"
       alt={alt}
+      //@ts-ignore
```

Comment:
use <any> above rather than ignoring here. and why useRef isn't used for storing this ref?